source/_integrations/wemo.markdown

```diff
@@ -72,6 +72,26 @@ Note that if you use static device entries, you may want to set up your router (
 
 If the device doesn't seem to work and all you see is the state "unavailable" on your dashboard, check that your firewall doesn't block incoming requests on port 8989, since this is the port to which the WeMo devices send their updates.
 
+### Device Options
+
+Clicking the `Configure` button on the WeMo integration will bring up some additional options that can be configured for WeMo devices.
+
+![Device Options](/images/integrations/wemo/device_options.png)
+
+**Subscribe to device local push updates**: WeMo devices support both the *Local Push* and *Local Polling* [IoT classes](https://www.home-assistant.io/blog/2016/02/12/classifying-the-internet-of-things/#classifiers). By default Home Assistant will subscribe to event notifications from WeMo devices and use the Local Push IoT class. As a fallback, if the Local Push doesn't work, Home Assistant will use Local Polling. Some devices are known not to work well with Local Push. WeMo devices expect to be on the same subnet as Home Assistant and will not work with Local Push otherwise. For devices that are known not to work with Local Push, the *Subscribe to device local push updates* option can be disabled to force only Local Polling to be used.
```

Comment:
```suggestion
**Subscribe to device local push updates**: WeMo devices support both the *Local Push* and *Local Polling* [IoT classes](/blog/2016/02/12/classifying-the-internet-of-things/#classifiers). Home Assistant will default subscribe to event notifications from WeMo devices and use the Local Push IoT class. If the Local Push doesn't work, Home Assistant will use Local Polling as a fallback. Some devices are known not to work well with Local Push. WeMo devices expect to be on the same subnet as Home Assistant and will not work with Local Push otherwise. For devices known not to work with Local Push, the **Subscribe to local push updates** option can be disabled to force only Local Polling to be used.
```